installer/install.iss

```diff
@@ -604,10 +603,12 @@ var
 // Returns true if at least one FSMonitor daemon was shut down successfully
 function ShutdownFSMonitorDaemons():Boolean;
 var
+    StdOut:String;
     FindRec:TFindRec;
-    ExitCode:DWORD;
-    Path,Str:String;
+    ExitCode:Integer;
+    Path:String;
```

Comment:
I removed `Str`, realised that `TExecOutput.StdOut` and `TExecOutput.StdErr` aren't `String`, but `TArrayOfString` and had to reintroduce a String here. I should probably at least re-join the lines here.